Automattic/wp-calypso
# Plans: Remove Google analytics feature page

- Resolves #8288
- Removes GA feature page
- Changes working route in plugins to `/plans/{siteSlug}/?feature=google-analytics`
- Cleans up dead code from contextualGoogleAnalyticsNudge test
- Removes remaining GA feature page scaffolding

## Testing

- Go to http://calypso.localhost:3000/plugins/{site}
- Click "Google Analytics" in business section
- You should be directed to /plans with GA row highlighted
- Go to http://calypso.localhost:3000/plans/features/google-analytics/:site
- See that you are redirected to /plans with the GA row highlighted

CC @lamosty @gwwar @rralian @retrofox
